charts/catalog/templates/apiserver-deployment.yaml

```diff
@@ -58,6 +58,8 @@ spec:
         {{- if not .Values.apiserver.auth.enabled }}
         - --disable-auth
         {{- end }}
+        - --feature-gates
+        - {{ .Values.apiserver.featureGates }}
```

Comment:
Seems like it might be good to wrap this in something like `if .Values.setFeatureGates`

Reply:
Done.